compiler-core/src/parse.rs

```diff
@@ -3606,7 +3525,9 @@ pub fn make_call(
                 if name != "_" {
                     return parse_error(
                         ParseErrorType::UnexpectedToken {
+                            found: ParserErrorCause::Descriptor("a name".into()),
```

Comment:
We can get rid of the `ParserErrorCause` enum by making a token here!
```suggestion
                            found: Token::Name(name),
```

Reply:
Oh, man! This was so obvious... thanks for pointing that out!